text/0008--individual-agency-company-pages.md

```diff
@@ -0,0 +1,131 @@
+- Start Date: 2018-07-24
+- RFC PR: 
+- Gatsby Issue: 
+
+# Summary
+
+As a follow-up to the [Site Showcase](https://next.gatsbyjs.org/showcase/), we want to create individual & organization pages highlighting the individual community members and what they've built. 
+
+We also want to create a free Job Board highlighting opportunities for employment building Gatsby websites at agencies and companies using Gatsby!
+
+# Motivation
+
+Some goals for the different components include:
+
+#### Individuals Pages
+* Highlight contributions made by community members (plugins built, code PRs)
+* "Official" landing pages to help community members in their personal website marketing efforts
+* Allow companies interested in building Gatsby sites to find individuals nearby them who are open for Gatsby projects
```

Comment:
what about not nearby them? I assume there could be a way to search not based on geolocation?

Reply:
Yes, the filter should definitely be optional, eg https://www.meteor.com/partners